### PR TITLE
Add accounts and persons to JSON API verifier.

### PR DESCRIPTION
**Pull request by nickygerritsen**
For now, don't drop team-members yet.

This should make https://github.com/DOMjudge/domjudge/pull/1513 pass.

### CODE REVIEW

File: contest-api/check-api.sh [hunk 1]
```diff
@@ -161,6 +165,7 @@ if [ -z "$API_URL" ]; then
 fi
 
 TMP=$(mktemp -d)
+echo $TMP
```
Comment (eldering) [inline contest-api/check-api.sh:168]: I assume this is a debugging leftover? If not, then use `verbose` to print it, so that it can be silenced.

**Comment by eldering** [comment]
There's a typo in the commit message and MR title.

File: contest-api/check-api.sh (hunk 1 above)
Comment (nickygerritsen) [reply contest-api/check-api.sh:168]: Lol yes my bad.

**Comment by nickygerritsen** [comment]
> There's a typo in the commit message and MR title.

Fixed.

**Review by eldering — APPROVED** [review]

**Pull request merged**